core/dbt/contracts/graph/nodes.py

```diff
@@ -1073,6 +1074,9 @@ class UnitTestDefinition(NodeInfoMixin, GraphNode, UnitTestDefinitionMandatory):
     config: UnitTestConfig = field(default_factory=UnitTestConfig)
     checksum: Optional[str] = None
     schema: Optional[str] = None
+    created_at: float = field(default_factory=lambda: time.time())
+    versions: Optional[UnitTestNodeVersions] = None
+    version: Optional[NodeVersion] = None
```

Comment:
What is this version for?

Reply:
Right now it's used to construct the name that's displayed on the terminal. But it also seems like we should have some way of distinguishing the related version for the unit test.